src/client/Microsoft.Identity.Client/Platforms/iOS/SystemWebview/ASWebAuthenticationPresentationContextProviderWindow.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AuthenticationServices;
+using Foundation;
+using UIKit;
+
+namespace Microsoft.Identity.Client.Platforms.iOS.SystemWebview
+{
+    internal class ASWebAuthenticationPresentationContextProviderWindow : NSObject, IASWebAuthenticationPresentationContextProviding
+    {
+        public UIWindow GetPresentationAnchor(ASWebAuthenticationSession session)
+        {
+            return UIApplication.SharedApplication.KeyWindow;
```

Comment:
The param session is not used? Can you add a comment on why this is / and what is the value that is returned?